analysis/build.gradle

```diff
@@ -93,14 +94,22 @@ subprojects {
     }
 
     kotlin {
-        jvmToolchain(8)
+        jvmToolchain(11)
     }
 
     kapt {
         arguments {
             arg("project", "${project.group}/${project.name}")
         }
     }
+
+    if("${project.name}".contains("RawTextParser")) {
```

Comment:
Why is the path differnt for this parsre?

Reply:
There is an error during the sonar analysis run, which states, that it can't find the test results from the RawTextParser subproject. I am trying different methods of configuration, but none seem to make a difference

Reply:
This didn't work either, so it will be changed next push.